### ISSUE TITLE
Upload list of recipients. Lang attribute is not set for parts of the page that are in different languages 

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#123]
### Description of issue

The default variable for email addresses cannot be localized (to my knowledge). We should set the lang attribute to the header of this column to English for better screen reader support. 

<img width="1065" alt="Capture d’écran, le 2021-10-19 à 13 17 00" src="https://user-images.githubusercontent.com/5230720/137960161-fa045e83-78e8-4166-bc9c-ce7b58cfe24e.png">

### Potential fix

- add `lang="en"` to the table header cell in the "email address" column

### Resources
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H57

**Comment by amazingphilippe** [comment]
Similar failures with the python test.

Failure output:

```
         for index, expected_item in enumerate(expected_items):
>           assert normalize_spaces(all_page_items[index].text) == expected_item
E           AssertionError: assert 'Select the f...one 2 folders' == 'folder_one 2 folders'
E             - Select the folder_one folder folder_one 2 folders
E             + folder_one 2 folders
```
